src/components/vote/DelegatesListItem.tsx

```diff
@@ -20,6 +19,8 @@ export const DelegatesListItem = ({
     onSelected: (delegate?: string) => void;
 }) => {
     const { t } = useTranslation();
+    const username = delegate.username() || '';
+    const isLongAddress = username.length > 16;
```

Comment:
issue with this approach is that you make the truncation dynamic through css, but the tooltip check is static. This can cause scenarios where the tooltip shows but the name is not truncated:
<img width="200" alt="image" src="https://github.com/ArdentHQ/arkconnect-extension/assets/35610748/43b62c20-cbd6-4051-acc6-a796f400a978">